src/python-common/ceph/deployment/drive_group.py

```diff
@@ -199,6 +199,9 @@ def __init__(self,
         #: the same node. See :ref:`orchestrator-osd-replace`
         self.osd_id_claims = osd_id_claims
 
+        #: Set the unmanaged flag (if deployment happens unattended)
+        self.unmanaged = unmanaged
```

Comment:
Thanks, the flag was not initialized before. Either creating the instance from new or from_json.

Reply:
fixed